src/javascript/app/ng-wig/ng-wig-plugin-adapter.component.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('component: ngWigPlugin', () => {
+    let component;
+    let scope;
+    let plugin;
+    let execCommand;
+    let editMode;
+    let disabled;
+    let options;
+    let content;
+    let element;
+    let compile;
+
+    beforeEach(module('ngWig'));
+
+    beforeEach(inject((_$componentController_, _$rootScope_, _$compile_) => {
+        scope = _$rootScope_.$new();
+        compile = _$compile_;
+
+        plugin = {
+            name: 'myPlugin',
+            pluginName: 'my-plugin'
+        };
+        editMode = false;
+        disabled = false;
+        options = ['Option 1', 'Option 2'];
+        content = 'Fake text';
```

Comment:
In this case, that we have only one complex test (`it('should replace with the plugin template'...`) we can. But for clarity, I would recommend to use beforeEach for initializing variables.